packages/ember-metal/lib/computed.js

```diff
@@ -382,49 +393,65 @@ ComputedPropertyPrototype.get = function(obj, keyName) {
   @return {Object} The return value of the function backing the CP.
   @public
 */
-ComputedPropertyPrototype.set = function computedPropertySetWithSuspend(obj, keyName, value) {
-  var oldSuspended = this._suspended;
+ComputedPropertyPrototype.set = function computedPropertySetEntry(obj, keyName, value) {
+  if (this._readOnly) {
+    throw new EmberError(`Cannot set read-only property "${keyName}" on object: ${inspect(obj)}`);
+  }
 
-  this._suspended = obj;
+  if (!this._setter) {
+    return this.clobberSet(obj, keyName, value);
```

Comment:
@cibernox - There was an RFC for that (and you have an open PR doing this IIRC). Sadly, it is too late in the beta cycle to add now.